packages/backend/src/routes/ticket.js

```diff
@@ -89,7 +90,7 @@ router.get('/sale-charge/:id', async (req, res, next) => {
   const ret = {
     price: ticket.price,
     fivePercent: ticket.price * 0.05,
-    shipping: ticket.deliveryMethod, // NEEDS METHOD, use distance
+    shipping: await getDeliveryMethod(req.params.id), // NEEDS METHOD, use distance
```

Comment:
Delivery method will be supplied in req.params. The delivery method in the database will not be updated until after the ticket is purchased. This 'sale-charge' route will be used before purchase. Also, please move the logic to the lib folder, because we will need to call the function again upon purchase.